pages/reference-modules/passkey/v0.2.1/signatures/verifySignatureAllowMalleability.mdx

```diff
@@ -0,0 +1,175 @@
+import { Tabs, Callout } from 'nextra/components'
+
+import LegacyCallout from '../../../../../components/callouts/LegacyCallout.mdx'
+import OnlySafeTxCallout from '../../../../../components/callouts/OnlySafeTxCallout.mdx'
+import ReentrancyCallout from '../../../../../components/callouts/ReentrancyCallout.mdx'
+import IrreversibilityCallout from '../../../../../components/callouts/IrreversibilityCallout.mdx'
+
+
+
+# `verifySignatureAllowMalleability` 
+
+
+
+Verifies the signature of a message using P256 elliptic curve, without signature
+malleability check.
+
+
+
+
+Defined in [` P256.sol`](https://github.com/safe-global/safe-modules/tree/passkey/v0.2.1/modules/passkey/contracts/libraries/P256.sol#L100)
+
+
+## Usage
+
+{/* <!-- vale off --> */}
+
+<Tabs items={['example.sol']}>
+  <Tabs.Tab>
+    ```solidity
+    interface ISafe {
```

Comment:
This is not a Safe interface, this is a library contract that is called on an `P256.Verifiers` value.

So, the correct usage is something like:

```solidity
using P256 for P256.Verifiers;

P256.Verifiers verifiers = ...;
verifiers.verifySignatureAllowMalleability(...);
```